docs/products/add-data-to-grid-datastores/README.md

```diff
@@ -17,6 +17,37 @@ Datastores can be attached to Runs or Sessions and preserve the file format and
 We don't charge for data storage!
 :::
 
+## Datastores with Private S3 Buckets
+
+At this time we are only supporting usage of private S3 buckets as datastores for BYOC users. You can grant Grid access to your desired buckets by following the official aws [documentation](https://aws.amazon.com/premiumsupport/knowledge-center/cross-account-access-s3/).
+
+As a convenience we provide a bucket policy that grants Grid access to all the contents of your specified bucket.
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::aws-account-id-associated-with-byoc:root"
+            },
+            "Action": [
+                "s3:ListBucket",
+                "s3:GetObject",
+                "s3:GetBucketLocation"
+            ],
+            "Resource": [
+               "arn:aws:s3:::your-bucket/*",
+               "arn:aws:s3:::your-bucket"
+           ]               
+        }
+    ]
+}
+```
+
+### Known limitations:
+
+Currently, we cannot guarantee the privacy of datastores connected to a public grid cloud account. It is on the roadmap and will be implemented in a future release. There are limited circumstances under which testing of a private aws s3 bucket is possible on the grid public cloud. Please reach out to support@grid.ai if you are interested.
```

Comment:
> Currently, we cannot guarantee the privacy of datastores connected to a public grid cloud account. It is on the roadmap and will be implemented in a future release. There are limited circumstances under which testing of a private aws s3 bucket is possible on the grid public cloud.

This sounds clunky. Let's get some input from the docs writers

Reply:
sounds good. @essiequoi has been requested to review everything as well.

Reply:
I have a proposition for a better way to phrase this, but the current question is whether we should even mention this at all or if we should restrict the use of this feature to BYOC users. We're discussing it on the Linear ticket, will report back

Reply:
i see this was removed, so ignore the above comment